artman/tasks/package_metadata_tasks.py

```diff
@@ -94,12 +95,42 @@ def _write_yaml(self, config_dict, dest):
         with open(dest, 'w') as f:
             yaml.safe_dump(config_dict, f, default_flow_style=False)
 
-class ProtoPackageMetadataGenTask(task_base.TaskBase):
-    default_provides = 'proto_pkg_dir'
+class JavaGrpcPackageMetadataConfigGenTask(PackageMetadataConfigGenTask):
```

Comment:
Is it necessary to move these here, instead of leaving them in place? (Not a big deal, just a bit harder to see what has changed)